side-navigation.yaml

```
@@ -0,0 +1,178 @@
+- heading: Welcome
+  subheadings:
+    - title: Get started
+      url: /docs
+    - title: Building with Vanilla
+      url: /docs/building-vanilla
+    - title: Customising Vanilla
+      url: /docs/customising-vanilla
+    - title: What's new in {version}
+      url: /docs/whats-new
+    - title: Migrating to v3.0
+      url: /docs/migration-guide-to-v3
+- heading: Base elements
+  subheadings:
+    - title: Code
+      url: /docs/base/code
+    - title: Forms
+      url: /docs/base/forms
+    - title: Reset
+      url: /docs/base/reset
+    - title: Separators
+      url: /docs/base/separators
+    - title: Tables
+      url: /docs/base/tables
+    - title: Typography
+      url: /docs/base/typography
+- heading: Components
+  subheadings:
+    - title: Accordion
+      url: /docs/patterns/accordion
+    - title: Badge
+      url: /docs/patterns/badge
+    - title: Breadcrumbs
+      url: /docs/patterns/breadcrumbs
+    - title: Buttons
+      url: /docs/patterns/buttons
+    - title: Cards
+      url: /docs/patterns/card
+    - title: Chips
+      url: /docs/patterns/chip
+    - title: Contextual menu
+      url: /docs/patterns/contextual-menu
+    - title: Divider
+      url: /docs/patterns/divider
+    - title: Empty state
+      url: /docs/patterns/empty-state
+    - title: Grid
+      url: /docs/patterns/grid
+    - title: Heading icon
+      url: /docs/patterns/heading-icon
+    - title: Icons
+      url: /docs/patterns/icons
+    - title: Images
+      url: /docs/patterns/images
+    - title: Links
+      url: /docs/patterns/links
+    - title: List tree
+      url: /docs/patterns/list-tree
+    - title: Lists
+      url: /docs/patterns/lists
+    - title: Logo section
+      url: /docs/patterns/logo-section
+    - title: Updated information Matrix
```

Some small copy/paste issue here I guess:

```suggestion
    - title: Matrix
```